docker/images/n8n/README.md

```diff
@@ -1,111 +1,108 @@
-		- [Start with other Database](#start-with-other-database)
-			- [Use with PostgresDB](#use-with-postgresdb)
-	- [Passing Sensitive Data via File](#passing-sensitive-data-via-file)
-	- [Example Setup with Lets Encrypt](#example-setup-with-lets-encrypt)
-	- [Updating a running docker-compose instance](#updating-a-running-docker-compose-instance)
-	- [Setting Timezone](#setting-timezone)
-	- [Build Docker-Image](#build-docker-image)
-	- [What does n8n mean and how do you pronounce it?](#what-does-n8n-mean-and-how-do-you-pronounce-it)
-	- [Support](#support)
-	- [Jobs](#jobs)
-	- [Upgrading](#upgrading)
-	- [License](#license)
+  - [Key Capabilities](#key-capabilities)
+  - [Contents](#contents)
+  - [Demo](#demo)
+  - [Available integrations](#available-integrations)
+  - [Documentation](#documentation)
+  - [Start n8n in Docker](#start-n8n-in-docker)
+  - [Start n8n with tunnel](#start-n8n-with-tunnel)
+  - [Use with PostgreSQL](#use-with-postgresql)
+  - [Passing sensitive data using files](#passing-sensitive-data-using-files)
+  - [Example server setups](#example-server-setups)
+  - [Updating](#updating)
+    - [Pull latest (stable) version](#pull-latest-stable-version)
+    - [Pull specific version](#pull-specific-version)
+    - [Pull next (unstable) version](#pull-next-unstable-version)
+    - [Updating with Docker Compose](#updating-with-docker-compose)
+  - [Setting Timezone](#setting-timezone)
+  - [Build Docker-Image](#build-docker-image)
+  - [What does n8n mean and how do you pronounce it?](#what-does-n8n-mean-and-how-do-you-pronounce-it)
+  - [Support](#support)
+  - [Jobs](#jobs)
+  - [License](#license)
 
 ## Demo
 
-[:tv: A short video (< 4 min)](https://www.youtube.com/watch?v=RpjQTGKm-ok) that goes over key concepts of creating workflows in n8n.
+This [:tv: short video (< 4 min)](https://www.youtube.com/watch?v=RpjQTGKm-ok)  goes over key concepts of creating workflows in n8n.
 
 ## Available integrations
 
-n8n has 200+ different nodes to automate workflows. The list can be found on: [https://n8n.io/nodes](https://n8n.io/nodes)
+n8n has 200+ different nodes to automate workflows. A full list can be found at [https://n8n.io/nodes](https://n8n.io/nodes).
 
 ## Documentation
 
-The official n8n documentation can be found under: [https://docs.n8n.io](https://docs.n8n.io)
+The official n8n documentation can be found at [https://docs.n8n.io](https://docs.n8n.io).
 
-Additional information and example workflows on the n8n.io website: [https://n8n.io](https://n8n.io)
+Additional information and example workflows are available on the website at [https://n8n.io](https://n8n.io).
 
 ## Start n8n in Docker
 
+In the terminal, enter the following:
+
 ```bash
+docker volume create n8n_data
+
 docker run -it --rm \
  --name n8n \
  -p 5678:5678 \
- -v ~/.n8n:/home/node/.n8n \
+ -v n8n_data:/home/node/.n8n \
  docker.n8n.io/n8nio/n8n
 ```
 
+This command will download all required n8n images and start your container.
```

Comment:
This might be a bit confusing since this is only a single image. What do you think?